source/Government.cpp

```diff
@@ -151,8 +152,8 @@ void Government::Load(const DataNode &node)
 						reputationMin = numeric_limits<double>::lowest();
 				}
 			}
-			else if(key == "raid")
-				raidFleets.clear();
+			else if(key == "raid" || key == "raiders")
```

Comment:
`raid` is the one that is deprecated
```suggestion
			else if(key == "raiders" || key == "raid")
```